pkg/cluster/px/px.go

```diff
@@ -177,11 +183,41 @@ func NewPXClusterProvider(dockerRegistrySecret, kubeconfig string) (Cluster, err
 	if err != nil {
 		return nil, err
 	}
+	k8sOps := k8s.Instance()
+
+	// Detect the installedNamespace
+	namespaces, err := k8sOps.ListNamespaces()
+	if err != nil {
+		return nil, err
+	}
+
+	var installedNamespace string
+OUTER:
```

Comment:
nit: Labels should be named just like normal variables - camelcase

Reply:
renamed to lower case outer